### ISSUE TITLE
Pipeline with Vault doing LDAP auth to get a Bitbucket repository

### ISSUE THREAD

**Issue by LauJorge** [jenkinsci/configuration-as-code-plugin#1442]
Hi! I was wondering if it is possible to have a pipeline where you checkout a bitbucket repository. This repository is accesible with LDAP credentials. Can vault provide those secrets?

[jenkins-jira]: https://issues.jenkins-ci.org
[dashboard]: https://issues.jenkins.io/secure/Dashboard.jspa?selectPageId=18341
[contributing]: ../blob/master/docs/CONTRIBUTING.md
[compatibility]: ../blob/master/docs/COMPATIBILITY.md

### Your checklist for this issue

🚨 Please review the [guidelines for contributing][contributing] to this repository.
💡 To better understand plugin compatibility issues, you can [read more here][compatibility]

- [ ] Create an issue on [issues.jenkins-ci.org][jenkins-jira], set the component to the plugin you are reporting it against

- [ ] Before creating an issue on [Jenkins JIRA][jenkins-jira], check for [an existing one via dashboard][dashboard]

- [ ] Link to [Jenkins JIRA issue][jenkins-jira]

- [ ] Ensure [Jenkins JIRA issue][jenkins-jira] has the label `jcasc-compatibility`

- [ ] Link to plugin's GitHub repository

### Description

Please describe your issue here.

**Comment by LauJorge** [comment]
So I'm still unable to retrieve any secret that is not on a kv engine. What's the point of that? Just static values on a pipeline?

**Comment by LauJorge** [comment]
@jetersen any ideas? You must have read my question. Can't retrieve values from openldap or AWS secret engines

**Comment by jetersen** [comment]
This question does not belong in JCasC as I understand it you want to use hashicorp vault plugin to access a secret during pipeline execution?
Than the question belongs to hashicorp vault plugin.

See https://github.com/jenkinsci/hashicorp-vault-plugin#usage-via-jenkinsfile

**Comment by LauJorge** [comment]
Yes I want to access dynamic secrets during pipeline execution. Thanks for your answer. You may close this. I'll ask in the right place

**Comment by jetersen** [comment]
I can also just transfer it 😆
